src/alto-ui/helpers/__tests__/sum.test.js

```diff
@@ -0,0 +1,24 @@
+import sum from '../sum';
+
+describe('sum', () => {
+  it('is sum', () => {
+    expect(sum()).toBe(0);
+  });
+
+  it('returns sum of numbers array without second argument', () => {
+    expect(sum([1, 2, 3])).toBe(6);
+  });
+
+  it('returns sum of strings array without second argument', () => {
+    expect(sum(['a', 'b'])).toBe(0);
+  });
+
+  it('returns sum of strings and numbers array without second argument', () => {
+    expect(sum(['a', 'b', 1])).toBe(1);
+  });
+
+  it('returns sum of object array', () => {
+    const mock = [{ width: 1 }, { width: 2 }, { width: 3 }];
+    expect(sum(mock, 'width')).toBe(6);
+  });

```

Comment:
Could you add a `it` without int ? 
`const mock = [{ width: '1' }, { width: '2' }, { width: '3' }];`